server/cluster/cluster.go

```diff
@@ -681,6 +713,9 @@ func (c *RaftCluster) processRegionHeartbeat(region *core.RegionInfo) error {
 	if c.regionStats != nil {
 		c.regionStats.Observe(region, c.getRegionStoresLocked(region))
 	}
+	for _, item := range expiredItems {
+		c.hotStat.Update(item)
```

Comment:
Will it have a problem since there is another place that may update at the same time?

Reply:
fixed.